rel/i18n/emqx_schema.hocon

```diff
@@ -679,13 +679,45 @@ common_ssl_opts_schema_verify.label:
 """Verify peer"""
 
 common_ssl_opts_schema_partial_chain.desc:
-"""Enable or disable peer verification with partial_chain"""
+"""Enable or disable peer verification with partial_chain:
+- `false`
+- `true`
+- `cacert_from_cacertfile`
+- `two_cacerts_from_cacertfile`
+When local verifies a peer certificate during the x509 path validation
+process, it constructs a certificate chain that starts with the peer
+certificate and ends with a trust anchor.
+By default, if the setting is set to `false`, the trust anchor is the
+rootCA, and the certificate chain must be complete.
+However, if the setting is set to `true` or `cacert_from_cacertfile`,
+the last certificate in the cacertfile will be used as the trust anchor
+certificate (such as an intermediate CA). This creates a partial chain
+in the path validation.
+Alternatively, if the setting is set to `two_cacerts_from_cacertfile`,
+one of the last two certificates in the cacertfile will be used as the
+trust anchor certificate, forming a partial chain. This option is
+particularly useful for CA certificate rotation.
+However, please note that it incurs some additional overhead, so it
+should only be used for certificate rotation purposes."""
```

Comment:
if the description cannot be split to bullets (one for each enum symbol), then there is no need to repeat the enums as is.

also some refine suggestions
```suggestion
"""Enable or disable peer verification with partial_chain.

When local verifies a peer certificate during the x509 path validation
process, it constructs a certificate chain that starts with the peer
certificate and ends with a trust anchor.
By default, if it is set to `false`, the trust anchor is the
Root CA, and the certificate chain must be complete.
However, if the setting is set to `true` or `cacert_from_cacertfile`,
the last certificate in `cacertfile` will be used as the trust anchor
certificate (intermediate CA). This creates a partial chain
in the path validation.
Alternatively, if it is configured with `two_cacerts_from_cacertfile`,
one of the last two certificates in `cacertfile` will be used as the
trust anchor certificate, forming a partial chain. This option is
particularly useful for intermediate CA certificate rotation.
However, please note that it incurs some additional overhead, so it
should only be used for certificate rotation purposes."""
```